src/core/state.go

```diff
@@ -42,13 +42,19 @@ const (
 type pendingTask struct {
 	Label    BuildLabel // Label of target to parse
 	Dependor BuildLabel // The target that depended on it (only for parse tasks)
-	Type     TaskType
+	Type     taskType
 }
 
 func (t pendingTask) Compare(that queue.Item) int {
 	return int((t.Type & priorityMask) - (that.(pendingTask).Type & priorityMask))
 }
 
+// A LabelPair is the type returned for parse tasks
+type LabelPair struct {
+	Label, Dependor BuildLabel
```

Comment:
Dependor is not good english, use dependee or dependent depending on which you mean here (or even go as far as DependedOnBy or DependsOn)

Reply:
According to Wiktionary it's the first-person singular present passive indicative of dependo, so maybe it's good Latin instead...